src/sql/workbench/contrib/connection/browser/connection.contribution.ts

```diff
@@ -128,10 +128,28 @@ configurationRegistry.registerConfiguration({
 			'description': localize('sql.defaultEngineDescription', "Default SQL Engine to use. This drives default language provider in .sql files and the default to use when creating a new connection."),
 			'default': 'MSSQL'
 		},
+		'sql.defaultAuthenticationType': {
```

Comment:
This is the new configuration setting.  Users can edit and set this enum.  via the ADO UI.